job-server/test/spark.jobserver/WebApiDataRoutesSpec.scala

```diff
@@ -0,0 +1,59 @@
+package spark.jobserver

```

Comment:
I think the name of the file should be DataRouteSpec.scala.

There are tools which helps us to jump to corresponding scala source to spec source. And filename helps there.